api/src/messaging/consumers.py

```diff
@@ -3,7 +3,8 @@
 from django.core.exceptions import ObjectDoesNotExist
 import json
 from .models import Room, Message
-
+from django.contrib.contenttypes.models import ContentType
+from asgiref.sync import sync_to_async
```

Comment:
I don't think we need to use sync_to_async. We can use the database_sync_to_async wrapper that is already imported